common/source/docs/common-brushless-escs.rst

```diff
@@ -1,121 +1,94 @@
+[/site]
 
-Configuration
-=============
+The frame rate of these pulses is usually between 50Hz to 490Hz. The faster frame rates allow quicker control reactions to be sent to the motor, if the ESC has capability for those frame rates.
 
-Protocol Selection
-------------------
-The ESC control protocol is selected by the :ref:`MOT_PWM_TYPE <MOT_PWM_TYPE>` parameter on Copter and Rover, or :ref:`Q_M_PWM_TYPE <Q_M_PWM_TYPE>` on QuadPlanes for its copter function motors.
+[site wiki="copter"]
+- Set :ref:`RC_SPEED <RC_SPEED>` to change the frame rate (default is 490hz)
+[/site]
+[site wiki="rover"]
+- Set :ref:`SERVO_RATE <SERVO_RATE>` to change the frame rate (default is 50hz)
+[/site]
+[site wiki="plane"]
+- Set :ref:`SERVO_RATE <SERVO_RATE>` to change the frame rate for forward motors (default is 50hz)
+- Set :ref:`Q_RC_SPEED <Q_RC_SPEED>` to change the frame rate for the VTOL esc/motors (default is 490hz)
+[/site]
 
-.. note:: The autopilot should be re-booted after changing the protocol type.
+.. note:: be sure of the capabilities of your ESC before selecting a higher frame rate to avoid damage to the ESC.
 
-On Plane, all other motors use Normal (PWM) protocol. However, in Plane, any motor, like the traditional fixed wing's main motor or Dual Motor Tailsitters (SERVOn_FUNCTION = 70 throttle, 73 throttle left and / or 74 throttle right), can be changed to a protocol other than PWM using the :ref:`SERVO_BLH_MASK<SERVO_BLH_MASK>` parameter to specify the output number of the motor together with the :ref:`SERVO_BLH_OTYPE<SERVO_BLH_OTYPE>` parameter to select the protocol of these motors independently of that selected for the copter function motors. 
+OneShot
+=======
 
-.. note:: All mask-based configuration can only be changed at a PWM group level, please consult the documentation for your flight controller to ascertain which outputs are on different groups. See :ref:`Mixing ESC Protocols<mixing-escs>` section below.
+OneShot (not to be confused with "OneShot125") is an older protocol that uses the same pulse widths as Normal PWM but has a fixed frame rate equal to the autopilot main loop rate.  There is little advantage for using this protocol over regular PWM.
 
-.. note:: In Plane, Oneshot (not Oneshot125) loop synchronization can be added to any motor running an normal PWM ESC using the :ref:`ONESHOT_MASK<ONESHOT_MASK>` bitmask. It will trigger pulses at the :ref:`SCHED_LOOP_RATE<SCHED_LOOP_RATE>`, but no lower than at 250Hz. Be sure the ESCs can handle this rate.
+[site wiki="copter, rover"]
+- Set :ref:`MOT_PWM_TYPE <MOT_PWM_TYPE>` to 1 (OneShot) and reboot the autopilot
+[/site]
+[site wiki="plane"]
+Oneshot (not Oneshot125) loop synchronization can be added to any motor running an normal PWM ESC using the :ref:`ONESHOT_MASK <ONESHOT_MASK>` bitmask. It will trigger pulses at the :ref:`SCHED_LOOP_RATE <SCHED_LOOP_RATE>`, but no lower than at 250Hz. Be sure the ESCs can handle this rate.
```

Comment:
```suggestion
Oneshot (not Oneshot125) loop synchronization can be added to any motor running an normal PWM ESC or Servo using the :ref:`ONESHOT_MASK <ONESHOT_MASK>` bitmask. It will trigger pulses at the :ref:`SCHED_LOOP_RATE <SCHED_LOOP_RATE>`, but no lower than at 250Hz. Be sure the ESCs can handle this rate.
```

Reply:
cool, added "servo".